src/App.js

```diff
@@ -121,6 +121,7 @@ export default class App extends Base {
 
     // Set the pubSub URL if present
     process.env.PUB_SUB_URL = new URLSearchParams(window.location.search).get('pubSubUrl');
+    process.env.MACADDRESS = new URLSearchParams(appUrl.search).get('macaddress');
 
     if (platform) {
```

Comment:
All variables passed in intent should be supported in URL - this part is not addressed

Reply:
also these have to be added https://github.com/rdkcentral/firebolt-certification-app?tab=readme-ov-file#supported-url-parameters

Reply:
fetching appId from url is also added